python/Quickstart.MD

```diff
@@ -142,192 +119,201 @@ Body of event request:
         "isSubmittedToPartner": false,
         "directSupplierImpact": {
             "businessImpact": "Business Impact",
-            "businessPriority": "HIGH"
-            }
+            "businessPriority": "MEDIUM"
         }
+    }
 }
+```  
+Note that your `False` values have been converted to `false` according to the JSON specification.  Everything looks good, so we can convert our dictionary to a JSON string.  
+
+**Type or copy the following into your Python terminal**  
+```
+request_body_json = json.dumps(request_body)
 ```
+Note that we have omitted the `indent=4, sort_keys=False` options from the `json.dumps()` command that we included in the print statement.  We used these for clarity when print, but are not necessary when creating our JSON string.
 
 ## Send Request
+Once your request header and request body are complete you are ready to submit your request.  
+You will need the URL for the APT-SCWM environment.  Contact your administrator for this information.  
```

Comment:
This requires a URL for the APT-SCWM environment. Contact your administrator for this information.

Reply:
updated